package.json

```diff
@@ -57,7 +59,8 @@
     "tslib": "^2.6.3",
     "turbo": "^2.0.9",
     "typescript": "5.5.3",
-    "vite": "5.3.3"
+    "vite": "5.3.3",
+    "run-script-os": "^1.1.6"
```

Comment:
👀

Reply:
This is necessary, because win need to have 'bash' before script name.
IDN why, because if it's outside of package.json script, i can do ./update_version and run this script.
I think it's issue with git bash console, it's unix console, but we don't have unix inside this script runner.
It's probably running in OS related compiler, i think so.

And this is a workaround, to users don't need to use update-version:Them_OS, they only calling update-version with param as default, and this package take responsibility of pick the right one script for them OS.

I hope you understand, what i was trying to explain 😆